test/helpers/bibucketNock.ts

```diff
@@ -0,0 +1,732 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import nock from 'nock';
+
+export class BitbucketNock {
+  user: string;
+  repoName: string;
+  url: string;
+
+  constructor(user: string, repoName: string) {
+    (this.user = user), (this.repoName = repoName);
+    this.url = 'https://api.bitbucket.org/2.0';
+  }
+
+  getApiResponse(resource: string, id?: number, value?: string) {
```

Comment:
What is a `value` here? I'd prefer to rename this argument to be more self-explaining.